From 3d59d5fac60f16b8bac87b6abd09073d432856a4 Mon Sep 17 00:00:00 2001
From: Jeremy Kerr <jeremy.kerr@au.ibm.com>
Date: Tue, 26 Aug 2014 09:49:21 +0800
Subject: [PATCH] Revert "occ: Increase wait time to 60s for OCC to init"

We don't have an OCC implementation yet, so keep the wait time low.

This reverts commit 793b06d73e1543350fc7c4c7c615c594ec1ddd62.

Signed-off-by: Jeremy Kerr <jk@ozlabs.org>
---
 hw/occ.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/occ.c b/hw/occ.c
index 62a62a8..7e6284a 100644
--- a/hw/occ.c
+++ b/hw/occ.c
@@ -90,7 +90,7 @@ static bool wait_for_all_occ_init(void)
 		 * homer_base+size before passing memory to host services.
 		 * This ensures occ_data->valid == 0 before OCC load
 		 */
-		tries = 600; /* 60 secs */
+		tries = 20; /* 2 secs */
 		while((occ_data->valid != 1) && tries--) {
 			time_wait_ms(100);
 		}
-- 
1.9.1

